councilmatic_core/templates/councilmatic_core/legislation.html

```diff
@@ -104,15 +104,24 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-users'></i> Sponsors</h3>
                         </thead>
                         <tbody>
                             {% for s in legislation.sponsorships.all %}
-                                {% if s.is_primary %}
+                                {% if s.primary %}
                                 <tr>
                                     <td>
                                         <div class="thumbnail-square">
-                                            <img src='{{s.person.headshot_url}}' alt='{{s.person.name}}' title='{{s.person.name}}' class='img-responsive img-thumbnail' />
+                                            {% if s.person %}
```

Comment:
may want to update this, depending on how we want to handle sponsorships without associated person objects.